gnosis/cowswap
# Price/Fee display UI change - Swap page and Confirmation Modal

# Summary

Blocked by #1086 

Closes the final price display changes in the wireframes here: https://www.figma.com/file/bcoZf5RIgVKDItdOaTZq18/GP-Swap?node-id=1310%3A2

SELL ORDER
![image](https://user-images.githubusercontent.com/21335563/127171906-1327224b-0a60-4eb7-a038-5e2533a037bb.png)


BUY ORDER
![image](https://user-images.githubusercontent.com/21335563/127171850-b9545cd7-d040-4e82-869c-32b983469874.png)

FEE > INPUT
![image](https://user-images.githubusercontent.com/21335563/127171963-d0f44b32-7d73-4202-a315-e0f710bef779.png)



  # To Test

1. make sell order
2. make buy order
3. input tiny amt of weth to see fee error also shows up
